CommonClient.py

```diff
@@ -720,6 +721,7 @@ async def process_server_cmd(ctx: CommonContext, args: dict):
             await ctx.send_msgs(msgs)
         if ctx.finished_game:
             await ctx.send_msgs([{"cmd": "StatusUpdate", "status": ClientStatus.CLIENT_GOAL}])
+        ctx.hint_points = args.get("hint_points", 0)
```

Comment:
Might be nicer to move this up near where other attrs are assigned rather than at the bottom after sent messages, like line 710/711.